src/async-wrap.cc

```diff
@@ -407,6 +407,40 @@ static void DisablePromiseHook(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+class AsyncWrap::DestroyParam {
+ public:
+  double asyncId;
+  v8::Persistent<Function> callback;
+};
+void AsyncWrap::WeakCallback(
+  const v8::WeakCallbackInfo<AsyncWrap::DestroyParam> &info
```

Comment:
We generally use left-leaning pointers and references, i.e. `… DestroyParam>& info`.